app/indexers/object_resource_indexer.rb

```diff
@@ -0,0 +1,18 @@
+# Generated via
+#  `rails generate hyrax:work ObjectResource`
+class ObjectResourceIndexer < Hyrax::WorkIndexer
```

Comment:
Do we need this boilerplate file?

Reply:
Thanks for the comment. Please refer to the above reply as they are the similar question.